nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -796,10 +801,10 @@ in
           '';
       };
 
-      proxyCache = mkOption {
-        type = types.submodule {
+      proxyCachePath = mkOption {
+        type = types.attrsOf (types.submodule ({ ... }: {
           options = {
-            enable = mkEnableOption (lib.mdDoc "Enable proxy cache");
+            enable = mkEnableOption (lib.mdDoc "This proxy cache path entry");
```

Comment:
it should be lowercase because `mkEnableOption` will do `Enable *description*`

Reply:
Fixed!